test/unit/plugins/rest_spec.js

```diff
@@ -797,18 +831,102 @@ describe("The Rest plugin", function () {
 				expect (failed, "concurrency").to.be.true;
 			});
 		});
+
+		describe("setting the status of an instance to a state not handled by update", function () {
+
+			var instanceID;
+			var responseCode;
+			var originalInstance;
+
+			before(function () {
+
+				return instances.createInstance()
+				.then(function (instance) {
+					instanceID = instance.id;
+					return instances.getInstance({ id : instance.id });
```

Comment:
What is the rationale for calling `instances.getInstance()` here? Does `instance.createInstance()` not return what you need?